packages/modules/user/src/services/user-module.ts

```diff
@@ -151,9 +157,7 @@ export default class UserModuleService
     const updates = invites.map((invite) => {
       return {
         id: invite.id,
-        expires_at: new Date().setMilliseconds(
-          new Date().getMilliseconds() + this.config.expiresIn * 1000
-        ),
+        expires_at: new Date(Date.now() + this.config.expiresIn * 1000),
```

Comment:
NOTE: this was giving a big int instead of a date, now it does the same thing but provide a date instead